src/cli/cmd/list.rs

```diff
@@ -286,3 +324,7 @@ impl CommandExecute for ListSubcommand {
         Ok(ExitCode::SUCCESS)
     }
 }
+
+fn string_has_whitespace(s: String) -> bool {
```

Comment:
Nit: AFAIK `chars()` doesn't require owning the `String`, so this could be `&str` to cut down on a `clone()` above.

Reply:
⬆️ George CLONEy over here 😄